src/Wallabag/CoreBundle/Import/PocketImport.php

```diff
@@ -81,18 +77,20 @@ public function getRequestToken($redirectUri)
     public function authorize($code)
     {
         try {
-            $response = $this->client->post('https://getpocket.com/v3/oauth/authorize', [], json_encode([
-                'consumer_key' => $this->user->getConfig()->getPocketConsumerKey(),
-                'code' => $code,
-            ]));
-        } catch (RequestException $e) {
+            $response = $this->client->request(Request::METHOD_POST, 'https://getpocket.com/v3/oauth/authorize', [
+                'json' => [
+                    'consumer_key' => $this->user->getConfig()->getPocketConsumerKey(),
+                    'code' => $code,
+                ],
+            ]);
+
+            $this->accessToken = $response->toArray()['access_token'];
+        } catch (ExceptionInterface $e) {
             $this->logger->error(sprintf('PocketImport: Failed to authorize client: %s', $e->getMessage()), ['exception' => $e]);
 
             return false;
         }
 
-        $this->accessToken = $this->jsonDecode($response)['access_token'];
-
         return true;
```

Comment:
Why don't you also move that line inside the `try`?

Reply:
no particular reason, makes sense so done now